api/asyncapi/websocket.yml

```diff
@@ -0,0 +1,88 @@
+asyncapi: "2.6.0"
+info:
+  title: Mainflux ws adapter
+  license:
+    name: Apache 2.0
+    url: https://www.apache.org/licenses/LICENSE-2.0
+  version: '1.0.0'
+  description: WebSocket API for sending messages through communication channels
+
+defaultContentType: application/json
+
+servers:
+  dev:
+    url: localhost:{port}
+    protocol: ws
```

Comment:
`WebSocket`, instead of `ws`.

Reply:
I will check on that

Reply:
From the documentation in [asyncapi.com](https://www.asyncapi.com/docs/tutorials/getting-started/servers) we would use `ws` to represent WebSocket protocol.